spec/requests/api/v1/rooms_spec.rb

```diff
@@ -143,4 +143,27 @@
       end
     end
   end
+
+  describe "GET /api/v1/rooms/popular" do
+    let(:margin) { 10 }
+    let(:entry_at) { Time.now.utc - margin }
+    let!(:room) { create(:public_room) }
+    let!(:user_room_log) { create(:user_room_log, room: room, exit_at: false) }
```

Comment:
__[RuboCop]__

RSpec/LetSetup: Do not use `let!` for test setup.


<!-- Comment by [SideCI](https://sideci.com) -->